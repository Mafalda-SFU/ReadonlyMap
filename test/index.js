const {expect, use} = require('chai');
const {test} = require('mocha');
const {jestSnapshotPlugin} = require('mocha-chai-jest-snapshot');

const ReadonlyMap = require('../index.js');


use(jestSnapshotPlugin());


test('layout', function()
{
  expect(ReadonlyMap).toMatchSnapshot();
})

test('no arguments', function()
{
  function func()
  {
    new ReadonlyMap
  }

  expect(func).to.throw();
})

test('argument', function()
{
  expect(new ReadonlyMap(new Map)).toMatchSnapshot();
})
