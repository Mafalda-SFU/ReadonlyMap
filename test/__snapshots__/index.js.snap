// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`argument 1`] = `Map {}`;

exports[`layout 1`] = `[Function]`;
